orchestra/urls.py

```diff
@@ -1,20 +1,19 @@
+from django.conf import settings
 from django.conf.urls import include
 from django.conf.urls import url
+from django.views.generic import RedirectView
 
 from orchestra.views import index
-from orchestra.views import OrchestraRegistrationView
 from orchestra.views import status
 
-
 urlpatterns = [
     url(r'^api/',
         include('orchestra.api_urls', namespace='orchestra')),
     url(r'^app/?', index, name='index'),
     url(r'^status/', status, name='status'),
-
-    # We have to override the register endpoint to emit the
-    # `orchestra_user_registered` signal
-    url(r'^accounts/register/$',
-        OrchestraRegistrationView.as_view(),
-        name='registration_register'),
+    url(r'^favicon.ico/$', RedirectView.as_view(
```

Comment:
there is always a 404 since the browser checks for the url `favicon.ico`, just made this change in product so thought it would be good to propagate